src/woo_publications/metadata/admin.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
+
+from ordered_model.admin import OrderedModelAdmin
+
+from .constants import InformatieCategorieOrigins
+from .models import InformatieCategorie
+
+
+@admin.register(InformatieCategorie)
+class InformatieCategorieAdmin(OrderedModelAdmin):
+    list_display = ("identifier", "naam", "order", "move_up_down_links")
+    readonly_fields = ("oorsprong",)
+    search_fields = (
+        "identifier",
+        "naam",
+    )
+    list_filter = ("oorsprong",)
+
+    def has_change_permission(self, request, obj=None):
+        if obj and obj.oorsprong == InformatieCategorieOrigins.value_list:
+            return False
+        return True
+
+    def get_readonly_fields(self, request, obj=None):
+        readonly_fields = self.readonly_fields
+        if obj and obj.oorsprong == InformatieCategorieOrigins.custom_entry:
+            readonly_fields += ("identifier",)
+
+        return readonly_fields
```

Comment:
I was also thinking of adding the editable flag but I thought that we maybe wanted to prevent it so we can still change it from the terminal.

Also why do we need a default value? Is there a legit argument to add that? It seems weird to create none existing urls.

Reply:
`editable=True` only controls the user interface, you can still change it through the ORM on the CLI.

> Also why do we need a default value? Is there a legit argument to add that? It seems weird to create none existing urls.

It's the cleanest way to achieve our desired behaviour - it serves both the API endpoints and admin at the same time at zero cost.

URLs do not necessarily need to be resolvable, they are often used for a unique namespace identifier. XML standards for example define namespaces as URLs that more often than not don't exist.

In this case, we only generate as URL to get a consistent shape of the data.

I'm suggesting a placeholder prefix now, in the future we could then easily find-and-replace that value if/when we have a more clever system and would actually allow resolving the URLs, but for now I want a fixed value so that we can easily move environments to different (test/acc/prod) domains.